mobile/app/src/main/java/org/exarhteam/iitc_mobile/IITC_Mobile.java

```diff
@@ -200,8 +247,11 @@ public boolean onEditorAction(final TextView v, final int actionId, final KeyEve
 
     @Override
     public void onSharedPreferenceChanged(final SharedPreferences sharedPreferences, final String key) {
-        if (key.equals("pref_force_desktop")) {
-            mDesktopMode = sharedPreferences.getBoolean("pref_force_desktop", false);
+		if (key.equals("pref_force_desktop")) {
+			mDesktopMode = sharedPreferences.getBoolean("pref_force_desktop", false);
```

Comment:
Wrong indentation

Reply:
Hmmm... my local copy shows that it is enabled, but diffs don't show it as being different than on the server...

Reply:
I just committed a new copy that should have a fixed indentation and the item enabled by default, but the change doesn't show in diffs on the server

Reply:
Looks like I finally got all those changes pushed to the server